src/components/LogView/LogLineText.js

```diff
@@ -13,7 +13,7 @@ type Props = {
   lineRefCallback: (?HTMLSpanElement, number, isUnmount?: boolean) => void,
   port: ?string,
   text: string,
-  highlightText: string[]
+  highlightText: string[],
```

Comment:
By convention JSON doesn't use trailing commas

Reply:
It isn't JSON though :)